ykman/cli/otp.py

```diff
@@ -242,11 +243,14 @@ def delete(ctx, slot, force):
 @click.option(
     '-G', '--generate-key', is_flag=True, required=False,
     help='Generate a random secret key. Conflicts with --key.')
+@click.option(
+    '--upload', is_flag=True, required=False,
+    help='Upload secret key to YubiCloud (opens in browser)')
```

Comment:
Terminology: Maybe we should use the word "credential" instead of secret key, since we are strictly speaking uploading more than the secret key?

Something like:
`Upload credential to YubiCloud (using a browser)`